spec/api-content-tracing-spec.js

```diff
@@ -115,6 +111,20 @@ describe('contentTracing', () => {
   describe('stopRecording', function () {
     this.timeout(5e3)
 
+    it('does not crash on empty string', (done) => {
+      const options = {
+        categoryFilter: '*',
+        traceOptions: 'record-until-full,enable-sampling'
+      }
+
+      contentTracing.startRecording(options).then(() => {
+        contentTracing.stopRecording('').then(path => {
+          expect(path).to.be.a('string').that.is.not.empty()
```

Comment:
please check with `fs.statSync` that this exists on the filesystem & is a file.
```suggestion
          expect(path).to.be.a('string').that.is.not.empty()
          expect(fs.statSync(path).isFile()).to.be.true()
```